### ISSUE TITLE
New install loads blank screen

### ISSUE THREAD

**Issue by mstevens-gc** [lindegroup/autopkgr#679]
Ran the installer 1.5.7 on 12.3.1 on Mac Studio. The installer appeared normal but the window is completely blank. Uninstalling and reinstalling have no effect. Went back and installed 1.5 with the same result after uninstalling 1.5.7. No errors during installation and was prompted for the helper as expected.

This is a brand new machine I was going to set up to handle all our jamf packages.

![Screen Shot 2022-04-11 at 5 56 48 PM](https://user-images.githubusercontent.com/90371262/162846665-065ef4e8-eab4-4367-ab91-ce556b43635e.png)
.

**Comment by 540aaron** [comment]
I have the same issue. New Studio out of box came with 12.2 installed. I upgraded to 12.3.1 and then formatted the SSD to try and see if a fresh install on 12.3.1 would fix it but the same blank screen issue persisted. Move aside the prefs and killall cfprefsd did not work. OP stated: _, I symlinked python to t python3 since python 2 is now deprecated and no longer installed. That also did not help anything._
